openage/convert/changelog.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+"""
+Asset version change log
+
+used to determine whether assets that were converted by an earlier version
+openage are still up to date.
+"""
+
+from ..log import info, warn
+from ..testing.testing import TestError
+
+
+# filename where to store the versioning information
+ASSET_VERSION_FILENAME = "asset_version"
+
+# available components for reconversion
+COMPONENTS = {
+    "metadata",
+    "graphics",
+    "sounds",
+}
+
+# each line represents changes to the assets.
+# the last line is the most recent change.
+CHANGES = (
```

Comment:
maybe it should be mandatory to add a comment for each line here? (comment should contain `git describe`, and some short description)